rest-server/core/src/main/scala/com/actionml/core/validate/JsonSupport.scala

```diff
@@ -123,4 +128,12 @@ trait JsonSupport extends LazyLogging {
   }
 
 
+  private val env = s"system.env.(.*)".r
+  private def enrichViaSystemEnv: String => String = {
+    case env(value) => sys.env.getOrElse(value, {
+      logger.warn(s"No ENV VAR for $value")
+      value
+    })
```

Comment:
@pferrel this ^^^ means silence (just warning in the logs), is it ok?

Reply:
Sounds good.